GiEnJul/Controllers/AdminController.cs

```diff
@@ -42,8 +42,14 @@ public async Task<IEnumerable<Giver>> GetGiversAsync()
             // return _mapper.Map<List<Models.Giver>>(await _giverRepository.GetAllAsync()).OrderBy(x => x.FullName).ToList();
         }
         [HttpGet("recipients")]
-        public async Task<IEnumerable<Recipient>> GetRecipientsAsync() {
-            return await _recipientRepository.GetAllAsModelAsync();
+        public async Task<List<Recipient>> GetRecipientsAsync() {
+            var recipients = await _recipientRepository.GetAllAsModelAsync();
+            foreach (var family in recipients)
+            {
+                var familyMembers = await _personRepository.GetAllByRecipientId(family.RowKey);
+                family.FamilyMembers.AddRange(familyMembers); 
+            }
```

Comment:
Small suggestion to change "family" to "recipient" in foreach, and assign the list as the new value instead of add to the empty list.
```suggestion
            foreach (var recipient in recipients)
            {
                recipient.FamilyMembers = await _personRepository.GetAllByRecipientId(recipient.RowKey);
            }
```
This could also be done with List.ForEach method combined with LINQ, but i do not know if it is equally fast or not. https://docs.microsoft.com/en-us/dotnet/api/system.collections.generic.list-1.foreach?view=netcore-3.1

Reply:
Changes implemented. Did not change to List.ForEach though, I think it looks more readable as it is :)